packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -14,19 +14,22 @@ import { useHasDomainsInCart } from '../hooks/has-domains';
 import Field from './field';
 import { SummaryLine, SummaryDetails, SummarySpacerLine } from './summary-details';
 import { LeftColumn, RightColumn } from './ie-fallback';
+import { prepareDomainContactDetails, isValid } from '../types';
 
 export default function WPContactForm( {
 	summary,
 	isComplete,
 	isActive,
 	CountrySelectMenu,
 	PhoneInput,
-	StateSelect,
 	countriesList,
+	renderDomainContactFields,
 } ) {
+	const translate = useTranslate();
 	const isDomainFieldsVisible = useHasDomainsInCart();
 	const contactInfo = useSelect( select => select( 'wpcom' ).getContactInfo() );
 	const setters = useDispatch( 'wpcom' );
+	const domainName = 'example.com'; // TODO: get the actual domain name
```

Comment:
```suggestion
	const [ items ] = useLineItems();
	const firstDomainItem = items.find( isLineItemADomain );
	const domainName = firstDomainItem ? firstDomainItem.sublabel : siteUrl;
```

For which you'll need to import `useLineItems` (from `@automattic/composite-checkout`) and `isLineItemADomain` (from `../hooks/has-domains`).